aiida/orm/nodes/repository.py

```diff
@@ -216,6 +216,14 @@ def walk(self, path: FilePath = None) -> Iterable[Tuple[pathlib.PurePosixPath, L
         """
         yield from self._repository.walk(path)
 
+    def copy_tree(self, target: Union[str, pathlib.Path], path: FilePath = None) -> None:
```

Comment:
The choice for the name of the argument `path` was taken because that is the name used in all other repository related methods where it concerns a relative path in the virtual hierarchy. That's why I decided to keep it the same for consistency.

Reply:
mmm ok, I still think it is a bit confusing when you have a target path and a source path that one of them is just arbitrarily named `path`, but I agree that if we already got into that problem with other methods we might as well stay consistent now. Maybe if/when we do the migration discussed below we could take this into account for the new interface.